vms/avm/vm.go

```diff
@@ -460,17 +460,21 @@ func (vm *VM) ParseTx(_ context.Context, bytes []byte) (snowstorm.Tx, error) {
 		return nil, err
 	}
 
+	err = rawTx.Unsigned.Visit(&txexecutor.SyntacticVerifier{
```

Comment:
Why not use `UniqueTx` inside of this?

Reply:
`UniqueTx` is being removed (also `SyntacticVerifier` is not taking in an interface, we can't use `UniqueTx` for that)